com.pot_app.pot.yml

```diff
@@ -0,0 +1,154 @@
+id: com.pot_app.pot
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: pot
+rename-icon: pot
+rename-desktop-file: pot.desktop
+
+finish-args:
+  - --socket=wayland
+  - --socket=x11
```

Comment:
it this support wayland, `fallback-x11` should be used insted of `x11`

Reply:
It supports Wayland, but it will crash if using ```fallback-x11``` instead of ```x11```

Reply:
> it this support wayland, `fallback-x11` should be used insted of `x11`

It supports Wayland, but it will crash if using fallback-x11 instead of x11

Reply:
then it doesn't support wayland.

Reply:
so the wayland socket should be removed.

Reply:
> so the wayland socket should be removed.

On Wayland:
```---socket=wayland``` works,
```-socket=wayland``` + ```--socket=x11``` works,
but ```--socket=wayland``` + ```--socket=fallback-x11``` crashes

Reply:
> then it doesn't support wayland.

No. When installing Pot via apt/dnf/pacman, it works flawlessly on Wayland.

Reply:
comparing apple and oranges. neither is sandboxed. Are you sure it's not using a XWayland?

Reply:
> comparing apple and oranges. neither is sandboxed. Are you sure it's not using a XWayland?

I am sure it's using Wayland.

Besides, I don't think both using ```--socket=wayland``` and ```--socket=x11``` is incorrect. [Nheko](https://github.com/flathub/io.github.NhekoReborn.Nheko/blob/master/io.github.NhekoReborn.Nheko.yaml) uses both ```--socket=wayland``` and ```--socket=x11```.

Reply:
two wrongs don't make a right.